roles/kubernetes-apps/helm/defaults/main.yml

```diff
@@ -18,3 +18,6 @@ helm_skip_refresh: false
 
 # Override values for the Tiller Deployment manifest.
 # tiller_override: "key1=val1,key2=val2"
+
+# Limit the maximum number of revisions saved per release. Use 0 for no limit.
+# helm_max_history: 0
```

Comment:
All variables is prefixed `tiller_` here. Could you do the same

Reply:
fixed in https://github.com/kubernetes-incubator/kubespray/pull/2894/commits/a8eca3c1dd85be312e85c2423a2d93277d7b683a